WordPress/src/main/java/org/wordpress/android/ui/uploads/UploadService.java

```diff
@@ -836,6 +819,43 @@ public void onMediaUploaded(OnMediaUploaded event) {
         }
     }
 
+    private boolean doFinalProcessingOfPosts() {
```

Comment:
I think it would be helpful to add a comment stating what the true/false result means.

Reply:
Totally - added in e14878b